src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -581,4 +581,13 @@
   <data name="required_property" xml:space="preserve">
     <value>required property</value>
   </data>
+  <data name="Error_colon_Could_not_complete_this_request" xml:space="preserve">
+    <value>Error: Could not complete this request.</value>
```

Comment:
Will the user get any sort of info more than this?  Can we say something like: "Check the output window for more details"?